doc/source/advanced-use.rst

```diff
@@ -249,3 +249,30 @@ Now simply create a jail and supply the pkgs.txt file:
 ``iocell create pkglist=/path-to/pkgs.txt tag=myjail``
 
 This will install ``nginx`` and ``tmux`` in the newly created jail.
+
+Using volumes
+-------------
+
+Jails can have shared directories from the host which is implemented using nullfs(5). The idea is based on volumes used in the docker world. Files can also be Volumes and are copied over at jail startup.
+
+The ``volumes`` property has the following syntax:
+    ::
+
+        /source/path(:/destination/path(:mount_options)) /another/source/path...
+
+``source_path`` is the minimum required input and causes ``destination_path`` to be the same as ``source_path``, but located within the jail's root path.
+
+All options are separated by a colon and multiple volumes can be specified by whitespace separation.
+All paths must be absolute and therefore start with a slash.
+Optionally comma separated ``mount_options`` can be passed onto the underlying mount command, mostly useful to specify read-only volumes. For files this is done by setting the immutable attribute after copying them into the jail.
+Volumes are only mounted during runtime.
+
+Volumes can be added at creation time:
+
+``iocell create volumes="/path/to/some/host/dir:/destination/within/jail/root /other/host/dir:/dest/in/jail:ro" tag=myjail``
```

Comment:
if you search for "host" in any of the documentation you will find it has always been used to refer to the jail host, not another host across the network, RTFM.
i can see how this may be a little more confusing in this case using volumes, hence why i changed it in the upcoming fix, but "host" it is definitely more consistent. i'm starting to question the usefulness of this fork to be honest. i doubt it will ever become mainstream and replace the original iocage in terms of use with this behaviour.